api/storage/elasticsearch/elasticsearch.go

```diff
@@ -85,24 +90,31 @@ func NewStorageManager(conf config.ElasticsearchConfig) (*StorageManager, error)
 	}
 
 	storageManager := &StorageManager{
-		client:       esclient,
-		defaultIndex: conf.Index,
+		client: esclient,
 	}
-	storageManager.createIndex(conf.Index)
+
+	storageManager.setCurrentIndexDay(time.Now().In(time.UTC))
```

Comment:
Does that mean that there's no correlation between the event's timestamp to the index they'll be written to?
What happens if there are different timezones and one of the collectors sent data of yesterday for instance?

Reply:
defer, we will wait until we have more data to solve this issue